libraries/AP_Scheduler/AP_Scheduler.tpp

```diff
@@ -0,0 +1,186 @@
+extern const AP_HAL::HAL& hal;
```

Comment:
Dear @dgrat where is the GPLv3 statement and @tridge statement?

Dear @dgrat your code style are seems good, but do you think that you are creating a new logic structure for Scheduler in AP_Scheduler.cpp file?

You are changing the only the code style and the renaming the file not the structure logic.